### DISCUSSION TITLE
is Swift embedded supported

### DISCUSSION THREAD

**Discussion by bastie** [SwiftPackageIndex/SwiftPackageIndex-Server#3214] [Q&A] [answered]
Hi,

last WWDC Swift embedded is presented. In result of looking for or do not looking for embedded packages: Support spi this package type or maybe as platform?

thx

**Comment by finestructure** [comment, marked as answer]
Swift embedded would be an interesting platform to test for at some point but at the moment it's still experimental and only available in development snapshots (as far as I know). We'll definitely keep an eye on it (and other new platforms) and add them to the test mix as soon as it's feasible.

**Reply by laosb** [reply]
Now Swift Embedded is in stable releases of Swift, this should be easier?

It would be very helpful if we can have a badge signaling whether this package can be built under embdedded environment.
